content/en/docs/concepts/workloads/_index.md

```diff
@@ -38,11 +38,13 @@ Kubernetes provides several built-in workload resources:
   `Pods` for that `StatefulSet`, can replicate data to other `Pods` in the same `StatefulSet`
   to improve overall resilience.
 * [`DaemonSet`](/docs/concepts/workloads/controllers/daemonset/) defines `Pods` that provide
-  node-local facilities. These might be fundamental to the operation of your cluster, such
-  as a networking helper tool, or be part of an
-  {{< glossary_tooltip text="add-on" term_id="addons" >}}.  
+  facilities that are local to a specific node.
   Every time you add a node to your cluster that matches the specification in a `DaemonSet`,
   the control plane schedules a `Pod` for that `DaemonSet` onto the new node.
+  Each pod in a `DaemonSet` performs a similar job to a system daemon on a classic Unix / POSIX
+  server. A `DaemonSet` might be fundamental to the operation of your cluster, such as
+  a plugin to run [cluster networking](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-network-model),
```

Comment:
cluster networking is not a good example, maybe log collection is a better alternative?

Reply:
Cluster networking solutions are mostly about CNI deployments. For a user, solutions like this are part of the Kubernetes itself, although not in the scope of "core Kubernetes". Once it is deployed there, you won't change it. The reason why I suggested log collection (as an example) is that we are talking about workload controllers. By "workload", I was trying to find something the user has more freedom to make choices.

This is a suggestion, not a blocker.

Reply:
Not sure if this is too much detail for the "workloads" index, but people also use Daemonsets to install software on the nodes.

Reply:
I'm still not clear why cluster networking is a bad example of a DaemonSet. I think it's the DaemonSet people most often encounter.

Reply:
Well, you may be correct. I cannot think of a better example for DaemonSet, i.e. something people want to run on almost every node.